compiler/rustc_ast_lowering/src/format.rs

```diff
@@ -148,7 +148,14 @@ fn make_format_spec<'hir>(
             None => sym::Unknown,
         },
     );
-    let flags = ctx.expr_u32(sp, placeholder.format_options.flags);
+    // This needs to match `FlagV1` in library/core/src/fmt/mod.rs.
+    let flags: u32 = ((placeholder.format_options.sign == Some(FormatSign::Plus)) as u32)
```

Comment:
exhaustively destructure `format_options` at the start of this function to make sure we handle all fields.

Reply:
Done!